windows/devices/uwb/UwbConnector.cxx

```diff
@@ -31,6 +31,10 @@ struct RegisteredCallbackToken
      */
     RegisteredCallbackToken(std::function<void(RegisteredCallbackToken*)> deregister) :
         Deregister([this, deregister = std::move(deregister)]() {
+            if (not deregister) {
+                PLOG_WARNING << "empty callback token";
+                return;
+            }

```

Comment:
When can/should this ever be the case?

Reply:
It shouldn't ever be the case but nothing prevents it from happening so I thought it would be better to log something if it is the case

Reply:
for example, if the deregister function was actually initialized with the default constructor std::function<>();